sky/tools/create_macos_framework.py

```diff
@@ -180,9 +195,9 @@ def process_framework(dst, args, fat_framework, fat_framework_binary):
         'FlutterMacOS.framework_.zip',
         'FlutterMacOS.framework.zip',
     ],
```

Comment:
yeah we would add codesign metadata here in the outer layer of the zip, something similar to  
         'zip',
        '-y',
        'FlutterMacOS.framework_.zip',
        'FlutterMacOS.framework.zip',
        'entitlements.txt',
        'without_entitlements.txt',

So adding  'entitlements.txt', 'without_entitlements.txt', on L197 and L198

Reply:
ack